gammapy/astro/population/simulate.py

```diff
@@ -126,21 +138,24 @@ def make_catalog_random_positions_sphere(size, center='Earth', distance=Quantity
     return table
 
 
-'''
 def make_cat_gauss_random(n_sources=100, glon_sigma=30, glat_sigma=1,
```

Comment:
It's commented out.
Don't update it here, we might remove it.

If you want, you can add a TODO or make an issue that these functions need to be removed or improved.
(no time to deal with it at the moment)